### PR TITLE
feat(auth): Support sms region config change on Tenant and Project level

### PR DESCRIPTION
**Pull request by pragatimodi**
PR for SMS Region
Original PR here : https://github.com/firebase/firebase-admin-node/pull/1673/files

### CODE REVIEW

File: test/integration/auth.spec.ts [hunk 1]
```diff
@@ -551,11 +551,11 @@ describe('admin.auth', () => {
             // each attempt). Occassionally, this call retrieves the user data
             // without the lastLoginTime/lastRefreshTime set; possibly because
             // it's hitting a different server than the login request uses.
-            let userRecord = null;
+            let userRecord: UserRecord | null = null;
 
             for (let i = 0; i < 3; i++) {
               userRecord = await getAuth().getUser('lastRefreshTimeUser');
-              if (userRecord.metadata.lastRefreshTime) {
```
Comment (renkelvin) [inline test/integration/auth.spec.ts:558]: maybe add a check to make sure `userRecord` is not `null` here.

**Review by renkelvin — COMMENTED** [review]
Mostly LGTM. Please remove the changes of `package-lock.json`.

**Review by renkelvin — APPROVED** [review]

**Comment by renkelvin** [comment]
Hey @lahirumaramba, we're releasing the SMS regions in the admin sdk given the launch is now approved.

Most of the changes were reviewed here: https://github.com/firebase/firebase-admin-node/pull/1673. It was merged then reverted.

Please take a look and feel free to let us know if there is any questions. Thanks!

**Review by lahirumaramba — APPROVED** [review]
Thank you @pragatimodi and @renkelvin! LGTM!
Merging this to the main branch will include the feature in the next SDK release... If we need more time to release this change we can merge this to a feature branch for now. Let me know.

**Comment by renkelvin** [comment]
> Thank you @pragatimodi and @renkelvin! LGTM! Merging this to the main branch will include the feature in the next SDK release... If we need more time to release this change we can merge this to a feature branch for now. Let me know.

Thanks @lahirumaramba ! Yeah, we'd like to include this feature in the next SDK release.

**Pull request merged**